src/topology/uniform_space/abstract_completion.lean

```diff
@@ -67,6 +68,11 @@ variables {α : Type*} [uniform_space α] (pkg : abstract_completion α)
 local notation `hatα` := pkg.space
 local notation `ι` := pkg.coe
 
+/-- If `α` is complete, then it is an abstract completion of itself -/
+def eq_itself_of_complete [separated_space α] [complete_space α] : abstract_completion α :=
```

Comment:
I am not a big fan of the name, I'd prefer something like `abstract_completion_self` or `abstract_completion_of_complete`

Reply:
Yeah, I am not a big fan of the name also. I'll go with ```abstract_completion.of_complete``` since I used the keyword ```complete``` when I was looking for this result.